### PR TITLE
Additional role endpoint has been added

### PR DESCRIPTION
**Pull request by AMcCartneyMTH**

### CODE REVIEW

File: src/infrastructure/organisations/api.js [hunk 1]
```diff
@@ -108,6 +108,8 @@ const getUserOrganisation = async (userId, organisationId, correlationId) => {
   return await callOrganisationsApi(uri, 'POST', payload, correlationId);
 };
 
+
```
Comment (malikimran34) [inline src/infrastructure/organisations/api.js:111]: spaces not required

File: src/app/services/serviceRoles.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const { extractPageParam, extractPageSizeParam } = require('../utils');
+const { listServiceRoles } = require('../../infrastructure/applications');
+
+const listServiceRoles = async (req, res) => {
+
+  const serviceRoles = await listServiceRoles(req.params.cid, req.correlationId);
+  return res.send({
```
Comment (malikimran34) [inline src/app/services/serviceRoles.js:7]: is logging required for cid or correlationId please?

File: src/infrastructure/organisations/api.js (hunk 1 above)
Comment (AMcCartneyMTH) [reply src/infrastructure/organisations/api.js:111]: These have been removed. Looks like they've been there a while.

File: src/app/services/serviceRoles.js (hunk 2 above)
Comment (AMcCartneyMTH) [reply src/app/services/serviceRoles.js:7]: I believe so. Logging is absent across much of the codebase. I have added logging now.

**Pull request merged**